config/pipeline.yaml

```
@@ -49,6 +49,22 @@ _anchors:
       compiler: gcc-10
       defconfig: x86_64_defconfig
 
+  kbuild-gcc-10-armel-android-mainline: &kbuild-gcc-10-armel-android-mainline-job
+    <<: *kbuild-job
+    image: kernelci/staging-gcc-10:armel-kselftest-kernelci
+    params: &kbuild-gcc-10-armel-params
+      arch: armel
+      compiler: gcc-10
+      defconfig: multi_v7_defconfig
+
+  kbuild-gcc-10-armel-android-mainline-imx: &kbuild-gcc-10-armel-android-mainline-imx-job
+    <<: *kbuild-job
+    image: kernelci/staging-gcc-10:armel-kselftest-kernelci
+    params: &kbuild-gcc-10-armel-imx-params
+      arch: armel
+      compiler: gcc-10
+      defconfig: imx_v6_v7_defconfig
```

```suggestion
    <<: *kbuild-gcc-10-armel-android-mainline-job
    params:
      <<: *kbuild-gcc-10-armel-params
      defconfig: imx_v6_v7_defconfig
```

Note: this will also inherit the rules when you add those, so no need to write them down twice ;)

---

right, I through it would append and not override, awesome, thanks